host/sgx/elf.c

```diff
@@ -1923,7 +1924,7 @@ oe_result_t elf64_load_relocations(
     {
         *size_out = oe_round_up_to_page_size(size);
 
-        if (!(*data_out = malloc(*size_out)))
+        if (!(*data_out = oe_memalign(OE_PAGE_SIZE, *size_out)))
```

Comment:
The line just immediately above rounds up to page size. Is this change redundant?

Reply:
The start address needs to be page aligned. The line above just ensures that the size is a multiple of OE_PAGE_SIZE.